src/test/java/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParseTreeTest.java

```diff
@@ -36,7 +36,9 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-public class JavadocParseTreeTest {
+import com.puppycrawl.tools.checkstyle.BaseCheckTestSupport;
+
+public class JavadocParseTreeTest extends BaseCheckTestSupport {
```

Comment:
It would be better to have a separate commit for refactoring/extending of `BaseCheckTestSupport`